card/mod/03_machines/lib/javascript/wagn_mod.js.coffee

```diff
@@ -241,6 +241,14 @@ $(window).ready ->
 #    $(this).closest('tr').find('.close-rule-link').click()
 
 
+  $('body').on 'click', '.submit-modal', ->
+    $(this).closest('.modal-content').find('form').submit()
+
+  $('body').on 'mouseover', '.closed-view', ->
+    $(this).find('.card-menu').show()
+  $('body').on 'mouseleave', '.closed-view', ->
+    $(this).find('.card-menu').hide()
+
```

Comment:
I think this can be done with just CSS.  

Actually, we may only have to delete a line:

````
.card-slot > .card-header .card-menu-link {
  display: none;
}
.card-slot.closed-view,
.card-slot:hover {
  > .card-header .card-menu-link {
    display: inline-block;
  }
}
````

(this is from bootstrap_cards.scss, and wikirate doesn't appear to have the latest version)